### DISCUSSION TITLE
Solved! How do I get 10bit encoding when using SVT-AV1

### DISCUSSION THREAD

**Discussion by Sray69** [staxrip/staxrip#902] [Q&A] [answered]
I know this is probably a dumb question but I am new to Staxrip. I am used to using FFmpeg/GUI's. The GUI's usually have a Custom Commands box that is easily found. I cannot find this in Staxrip. When I choose the SVT-AV1 encoder and then click on the Encoder Options, there is a "Custom:" input box but from what I can tell it is only for -svtav1-params and not ffmpeg.

I am trying to get 10bit encodes using SVT-AV1. I cannot find an option to set it in the encoder options so I figured I would try to set it via ffmpeg custom commands (-pix_fmt yuv420p10le). If there is a different/better way of doing this in Staxrip please let me know.

Thanks

UPDATE: I got the answer elsewhere.

**Comment by jkkma** [comment]
"nvm fixed :)"
Common courtesy would tell you to post how you fixed the problem for other people that have the same issue (me for example)

**Comment by Dendraspis** [comment, marked as answer]
@jkkma 
You have to add a filter to convert the source to 10bits, so if you're using 
- AviSynth, add `Color » Convert » ConvertBits » 10`
- VapourSynth, add `Color » Convert » BitDepth » 10`
